tests/pytest/eligibility/test_views.py

```diff
@@ -0,0 +1,57 @@
+from django.urls import reverse
+import pytest
+
+from benefits.core.models import TransitAgency
+from tests.pytest.conftest import with_agency
+
+
+def set_agency(mocker):
+    agency = TransitAgency.objects.first()
+    assert agency
+    with_agency(mocker, agency)
+    return agency
+
+
+def set_verifier(mocker):
+    agency = set_agency(mocker)
+
+    mock = mocker.patch("benefits.core.session.verifier", autospec=True)
+    verifier = agency.eligibility_verifiers.first()
+    assert verifier
+    mock.return_value = verifier
+
+
+@pytest.mark.django_db
+def test_index_with_agency(mocker, client):
+    agency = TransitAgency.objects.first()
+    assert agency
+    with_agency(mocker, agency)
```

Comment:
I think you could say `set_agency(mocker)` here to shortcut these 3 lines

Reply:
Good catch, thanks!